package.json

```diff
@@ -14,19 +18,27 @@
     ],
     "*.{scss,html,js}": [
       "yarn lint:copyright --fix"
+    ],
+    "*.{tsx,ts,jsx,js}": [
+      "prettier --write",
+      "node packages/configs/copyrightLinter.js --fix"
     ]
   },
   "scripts": {
     "build": "turbo run build",
     "dev": "turbo run dev",
-    "test": "turbo run test --",
+    "test": "turbo run test",
```

Comment:
Running `yarn test` also builds the playgrounds. Is this required?

![image](https://user-images.githubusercontent.com/45748283/208959678-7b5badf9-32e0-40d1-a1ef-47679387e004.png)

Reply:
that's interesting. this should not be happening because the playgrounds have no `test` script

Reply:
Also, I'm having a bit of trouble writing a turbo `--filter` to run visual tests in only the CSS repo and only for a specific component (like we used to do before with `yarn test --filter=information-panel`). Is it something like:

```
yarn test --filter=@itwin/itwinui-css -- --filter=information-panel
```

I'm not sure how to pass the filter to the underlying command and not to turbo.

Reply:
i dont think this will be possible unfortunately. we might be better off adding separate commands for `test:css`/`test:storybook` (in a future PR). for now you can use `yarn workspace @itwin/itwinui-css test --filter=information-panel` (after building).